decidim-elections/config/locales/en.yml

```diff
@@ -998,6 +1007,7 @@ en:
             banner_image: Banner image
             census_contact_information: Census contact information
             census_contact_information_help: This contact information is for a participant who wants to report issues with the census. It can be an email address, a contact form on another site, a survey for visitors, etc.
+            hide_can_i_vote_help: Whether to show the "Can I vote?" link at the public votings menu
```

Comment:
```suggestion
```
I can't make the suggestion as it's out of the PR, but around line 1003 this needs to be added: 

show_check_census_help: Whether to show the "Can I vote?" link at the public votings menu